README.md

```diff
@@ -212,7 +212,7 @@ evince $dest/*pdf
 In order to download input NETCDF data from S3, and S3 URI can be specified as the setup parameter `infile` as below:
```

Comment:
should we maybe mark this as example (not that someone thinks the whole cosmo-2e...MUE stuff is required)
```suggestion
In order to download input NETCDF data from S3, and S3 URI can be specified as the setup parameter `infile` as below (example):
```

Reply:
Applied the suggestion, thanks!